Editor/Scripts/Analytics/MeshsyncAnalytics.cs

```diff
@@ -48,12 +88,24 @@ private struct SyncEventData {
             public string entitySyncType;
         }
 
+
+        public MeshSyncAnalytics() {
+            this.syncEventStack = new Stack<TimedEvent<SyncEventData>>();
+            this.lastSync = DateTime.Now;
+
+            MinTimeBetweenSync = DEFAULT_TIME_SPAN;
+        }
+
+        [Conditional("DEBUG")]
         private static void logIfWarning(AnalyticsResult resp) {
             if (resp != AnalyticsResult.Ok) {
-                Debug.LogWarning($"Analytics endpoint reported: {resp} when should be {AnalyticsResult.Ok}");
+                UnityEngine.Debug.LogWarning($"Analytics endpoint reported: {resp} when should be {AnalyticsResult.Ok}");
```

Comment:
I thought we agreed to comment this out for now ?

Reply:
Using the [Conditional()] attribute, this function won't do anything unless that macro (now DEBUG_ANALYSIS) is defined at compile time. It was DEBUG but then I was told by the team that most developers work in Debug by default so it wouldn't change anything. Now in order to print anything you'd need to #define DEBUG_ANALYTICS in that file for the messages to appear.